css/style.css

```diff
@@ -0,0 +1,17 @@
+#navbar > ul > li:hover{
+  background: white;
+}
+
+.row{
+  margin-bottom: 20px;
+}
+
+.panel-default{
+  background-color: #f8f8f8;
+  border-color: #e7e7e7;
+}
+
+/* source: https://stackoverflow.com/questions/10591422/bootstrap-carousel-image-doesnt-align-properly */
```

Comment:
Don't leave commented-out code in the final product.

Reply:
What would be the correct way of sourcing where your code came from then?
Secondly - What about the carousel doesn't work? it's meant to scroll automatically with no buttons.